client/landing/jetpack-cloud/sections/backups/controller.js

```diff
@@ -19,6 +23,20 @@ export function wrapInSiteOffsetProvider( context, next ) {
 	next();
 }
 
+export function showUpsellIfNoBackup( context, next ) {
+	context.primary = (
+		<UpsellSwitch
+			UpsellComponent={ BackupsUpsell }
+			display={ context.primary }
+			getStateForSite={ getRewindState }
+			QueryComponent={ QueryRewindState }
+		>
+			<p>{ '...' }</p>
```

Comment:
Is this intentional?

Reply:
just a leftover from when I copied the scan example and didn't have a backups placeholder to put there. I will get rid of it 👍